static/src/js/schemas/uiSchemas/variables/measurementIdentifiers.js

```diff
@@ -0,0 +1,152 @@
+import CustomSelectWidget from '../../../components/CustomSelectWidget/CustomSelectWidget'
+import CustomTextWidget from '../../../components/CustomTextWidget/CustomTextWidget'
+
+const measurementIdentifiersUiSchema = {
+  MeasurementIdentifiers: {
+    'ui:title': 'Measurement Identifiers',
+    items: {
+      'ui:field': 'layout',
+      'ui:title': 'Measurement Identifier',
+      'ui:controlled': {
+        name: 'measurement_name',
+        map: {
+          MeasurementContextMedium: 'context_medium',
+          MeasurementObject: 'object'
+        },
+        clearAdditions: ['MeasurementQuantities.Value']
+      },
+      'ui:layout_grid': {
+        'ui:row': [
+          {
+            'ui:col': {
+              md: 12,
+              children: [
+                {
+                  'ui:row': [
+                    {
+                      'ui:col': {
+                        md: 12,
+                        controlName: 'context_medium',
+                        children: ['MeasurementContextMedium']
+                      }
+                    }
+                  ]
+                },
+                {
+                  'ui:row': [
+                    {
+                      'ui:col': {
+                        md: 12,
+                        children: ['MeasurementContextMediumURI']
+                      }
+                    }
+                  ]
+                },
+                {
+                  'ui:row': [
+                    {
+                      'ui:col': {
+                        md: 12,
+                        controlName: 'object',
+                        children: ['MeasurementObject']
+                      }
+                    }
+                  ]
+                },
+                {
+                  'ui:row': [
+                    {
+                      'ui:col': {
+                        md: 12,
+                        children: ['MeasurementObjectURI']
+                      }
+                    }
+                  ]
+                },
+                {
+                  'ui:row': [
+                    {
+                      'ui:col': {
+                        md: 12,
+                        children: ['MeasurementQuantities']
+                      }
+                    }
+                  ]
+                }
+
+              ]
+            }
+          }
+        ]
+      },
+      MeasurementContextMedium: {
+        'ui:widget': CustomSelectWidget
+      },
+      MeasurementContextMediumURI: {
+        'ui:widget': CustomTextWidget
+      },
+      MeasurementObject: {
+        'ui:title': 'Measurement Object',
+        'ui:widget': CustomSelectWidget
+      },
+      MeasurementObjectURI: {
+        'ui:widget': CustomTextWidget
+      },
+      MeasurementQuantities: {
+        'ui:title': 'Measurement Quantity',
```

Comment:
When I click on "Add Measurement Quantity" it throws an error: "getParentFormData.js:36 Uncaught TypeError: Cannot read properties of undefined (reading 'at')"

Reply:
Fixed